src/polyfill.js

```diff
@@ -55,6 +59,7 @@ function setup() {
           return context;
         },
         config: normalizedConfig,
+        getDefaultConfiguration: getDefaultConfiguration(),
```

Comment:
This is currently a bug - I haven't figured out how to make a static method for `Sanitizer`.

Right now, you can call `new Sanitizer().getDefaultConfiguration`. However, we want to be able to call `Sanitizer.getDefaultConfiguration`.

https://wicg.github.io/sanitizer-api/#dom-sanitizer-getdefaultconfiguration

Reply:
Thanks! That worked for me. I was setting `sanitizer.getDefaultConfiguration` before `window[GLOBALNAME] = sanitizer` last time I tried this 🤦‍♂️